iaso/api/setup_account.py

```diff
@@ -44,6 +44,7 @@ def create(self, validated_data):
             password=validated_data["password"],
             first_name=validated_data.get("user_first_name", ""),
             last_name=validated_data.get("user_last_name", ""),
+            is_superuser=True,
```

Comment:
This is unfortunately not the solution: this gives access to all the accounts through the admin and should NOT be given to an account admin.